packages/client/src/v2-events/features/users/cache.ts

```diff
@@ -19,7 +19,10 @@ import {
 import { findUserIdsFromDocument, findUserIdsFromIndex } from './utils'
 
 async function cacheUsers(userIds: string[]) {
-  const users = await trpcClient.user.list.query(userIds)
+  const { queryFn, ...options } =
+    trpcOptionsProxy.user.list.queryOptions(userIds)
```

Comment:
Looks like we need to not use `trpcClient` in most cases as it doesn't use the same query defaults (like queryFn) we want to trigger as part of for instance fetching the user list